packages/remix/src/generators/application/files/integrated/package.json__tmpl__

```diff
@@ -17,10 +17,14 @@
     "@types/react-dom": "<%= typesReactDomVersion %>",
     "eslint": "<%= eslintVersion %>",
     "typescript": "<%= typescriptVersion %>",
-    "vite": "<%= viteVersion %>",
+    "vite": "<%= viteVersion %>"
   },
   "engines": {
     "node": ">=20"
   },
-  "sideEffects": false
+  "sideEffects": false<% if (isUsingTsSolution) { %>,
+  "nx": {
+    "name": "<%= name %>"
```

Comment:
We need to add the rest of the metadata and not set the `name` if it matches the `name` property in the `package.json`.